### DISCUSSION TITLE
Airlock container access - no generate button

### DISCUSSION THREAD

**Discussion by bwiseman** [microsoft/AzureTRE#3622] [Q&A]
Sorry for having another question but I have the TRE up and running and am straightening things out in my head.

I have the base workspace and a windows 11 VM with no internet access.    I have imported a file that I want to make available to the windows vm.   The import and review worked ok but the simple Windows 11 doesn't have az or Storage Explorer installed.  

What am I missing?  How do I download the data into a windows vm after approval?  
I can't just paste the SAS url into a browser.    Linux has storage-explorer so do I need to hop through a shared drive?

**Comment by marrobi** [comment]
So most production deployments have a custom image with tooling installed. 

I often put storage explorer into a storage account that I can access as a workaround  for demos.

We should probably add the Microsoft downloads site to the package mirror.

**Comment by bwiseman** [comment]
Is there a beginners guide anywhere to building custom images?

**Reply by marrobi** [reply]
@martinpeck any chance you can share your repository? Thanks.

**Reply by martinpeck** [reply]
This might be a useful resource: https://github.com/martinpeck/custom-vm-sample-toolkit

I'll be updating this in the next week, so expect a few changes.

**Comment by bwiseman** [comment]
The report is on my local machine but I could zip it.  There's no generate button but it generates a new sas key automatically 

Barbars
